tests/WebAssertTest.php

```diff
@@ -1115,6 +1151,8 @@ protected function assertWrongAssertion($assertion, $arguments, $exceptionClass,
         try {
             call_user_func_array(array($this->assert, $assertion), $arguments);
             $this->fail('Wrong assertion should throw an exception');
+        } catch (PHPUnit_Framework_AssertionFailedError $e) {
```

Comment:
The `PHPUnit_Framework_AssertionFailedError` is thrown by `$this->fail(...)`. It makes not sense to investigate this exception in the catch block below, whose purpose it is to check whether the exception thrown by the line before (`call_user_func_array...`) is the expected one. If it is caught here, the error message shown by `assertWrongAssertion` is not 'Wrong assertion should throw an exception' but something like 'failed asserting that foo is instance of bar'. Therfore, I added this special catch block to let the exception thrown by fail pass through.